src/mongocxx/exception/private/mongoc_error.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 MongoDB Inc.
+// Copyright 2015 MongoDB Inc.
```

Comment:
Yes, this was not meant to be a rename. Can I explicitly tell git that it isn't?